### PR TITLE
Video embed omniture tracking

### PR DESCRIPTION
**Pull request by phamann**
This enables Omniture tracking for off-site embedded video on embed.theguardian.com.
- Add event binding to video-embed controller
- Bundle omniture and omnitureMedia in application
- Change `eVar11` property to `Embedded` for reporting
## Todo
- Make the event tracking/binding more DRY by porting to own module which can be shared between the main `media.js` 
- Add Ophan tracking
- Add secure endpoint

/cc @rich-nguyen @NathanielBennett

### CODE REVIEW

File: static/src/javascripts/bootstraps/video-embed.js [hunk 1]
```diff
@@ -106,9 +175,14 @@ define([
 
                 player.fullscreener();
 
+                deferToAnalytics.init();
```
Comment (rich-nguyen) [inline static/src/javascripts/bootstraps/video-embed.js]: this won't do anything useful, I think I meant somehow cause the module deferToAnalytics to be inited in such a way that it cannot miss the event it's listening for.

So you can remove the deferToAnalytics reference here

File: static/src/javascripts/bootstraps/media.js [hunk 2]
```diff
@@ -145,6 +145,7 @@ define([
     function bindContentEvents(player) {
         var events = {
             end: function () {
+                bindContentEvents(player);
```
Comment (rich-nguyen) [inline static/src/javascripts/bootstraps/media.js]: Rebind?

**Comment by rich-nguyen** [comment]
:+1:

**Comment by paperboyo** [comment]
Hello,

This isn't related as such, but it is about embeds and I just wanted to catch your attention, so please excuse the intrusion.

I think it's very important for embedded player to include the ability to link to our video page (pt. 14 https://github.com/guardian/frontend/issues/5237).

Ideally, it would be a vector logo shown at all times that when clicked would pause the play and get a user to the Guardian site in a new tab at exactly the same moment in the video (just like youtube works). The problem is, currently, we burn in the raster logo over our videos...

But maybe just a link along the one for an Embed or in a context menu would suffice for the start?

A recent case of that (very good!) video (http://www.theguardian.com/world/video/2015/jan/19/pretty-radical-young-woman-poland-far-right-video) embedded on plenty of Polish sites illustrates the problem well. Almost all polish media for some stupid/outdated reasons are reluctant to link to source articles. So even though our video is all over them, there is no way for the user to easily get to the Guardian site without resorting to Google search.

Just something to keep at the back of the head, I think.

Thanks for reading.

Regards
Mateusz

**Pull request merged**